src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -127,6 +130,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
     }
   }
 
+  private object IsCommand {
```

Comment:
Ah, alright. It'd be good to be consistent on that. But it is not that critical for now.